src/_includes/components/footer.njk

```diff
@@ -15,8 +15,11 @@
           <label for="newsletter-email">Email address</label>
           <div class="field--connected-btn">
             <input id="newsletter-email" type="email" name="email" 
-              placeholder="name@example.com" autocapitalize="off" required />
-            <input type="hidden" name="tags" value="Web Sign up"/>
```

Comment:
I mentioned in slack, but notice this `<input type="hidden" name="tags" value="Web Sign up"/>` has been removed. We need that, haha. Likely happened during a rebase

Reply:
Fixed, my bad 🤦‍♂️. Back to the drawing board with rebasing for me.